rules/validator/validator.go

```diff
@@ -121,12 +121,29 @@ func (v *validator) validateMappingRules(mrv []view.MappingRule) error {
 
 		// Validate the aggregation ID.
 		if err := v.validateAggregationID(rule.AggregationID, firstLevelAggregationType, types); err != nil {
-			return fmt.Errorf("mapping rule '%s' has invalid aggregation id %v: %v", rule.Name, rule.AggregationID, err)
+			return fmt.Errorf("mapping rule '%s' has invalid aggregation ID %v: %v", rule.Name, rule.AggregationID, err)
 		}
 
-		// Validate that the storage policies are valid.
-		if err := v.validateStoragePolicies(rule.StoragePolicies, types); err != nil {
-			return fmt.Errorf("mapping rule '%s' has invalid storage policies in %v: %v", rule.Name, rule.StoragePolicies, err)
+		// Validate the drop policy is valid.
+		if !rule.DropPolicy.IsValid() {
+			return fmt.Errorf("mapping rule '%s' has an invalid drop policy: value=%d, valid_values=%v",
```

Comment:
Nit: It might be nice to include the string version of the current policy in the error message.

Reply:
Sure thing.